licenses/publicsuffixes_mpl-2

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
[for reviewer] we need to embed a copy of the MPL 2.0 license as it's included through okhttp dependency.